cedar-policy/src/api.rs

```diff
@@ -59,6 +59,29 @@ use std::collections::{BTreeMap, BTreeSet, HashMap, HashSet};
 use std::io::Read;
 use std::str::FromStr;
 
+// PANIC SAFETY: `CARGO_PKG_VERSION` should return a valid SemVer version string
+#[allow(clippy::unwrap_used)]
+pub(crate) mod version {
+    use lazy_static::lazy_static;
+    use semver::Version;
+
+    lazy_static! {
+        // Cedar Rust SDK Semantic Versioning version
+        static ref SDK_VERSION: Version = env!("CARGO_PKG_VERSION").parse().unwrap();
+        // Cedar language version
+        // The patch version field may be unnecessary
+        static ref LANG_VERSION: Version = Version::new(4, 0, 0);
```

Comment:
Nit: Cedar lang version only uses MAJOR/MINOR, per https://docs.cedarpolicy.com/other/doc-history.html

Reply:
Yeah. The semver crate won't let me to specify optional patch number. Should we make a wrapper out of it?

Reply:
If it's not easy to change, then it's fine as-is.